### DISCUSSION TITLE
hello I have not received my github student benifits yet....please help as soon as possible

### DISCUSSION THREAD

**Discussion by Rajesh4434** [community/community#133827] [GitHub Education]
### Hi everyone,

hello I have not received my github student benifits yet....please help as soon as possible

**Comment by davevad93** [comment]
Hi @Rajesh4434 ,

If your application status display "`Congrats! Your request for @Rajesh4434 was approved and your benefits will be available soon.`" it means you have to wait, usually it takes a few days, but sometimes it can take up to a couple of weeks. The waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Rajesh4434 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470
